src/test/scala/CollaboratorOpSpec.scala

```diff
@@ -0,0 +1,13 @@
+import org.scalatest.path.FunSpec
+import codecheck.github.models.Collaborator
+import scala.concurrent.Await
+
+class CollaboratorOpSpec extends FunSpec with Constants {
+
+  describe("listCollaborators") {
+    it("should return atleast one collaborator") {
+      val res = Await.result(api.listCollaborators(organization, repo), TIMEOUT)
+      assert(res.length >= 1)
```

Comment:
Like this.

``` scala
val c = res(0)
assert(res.id > 0)
assert(res.login.length > 0)
assert(res.avatar_url.length > 0)
assert(res.url.length > 0)
assert(res.site_admin == false)
//This line cause error
assert(res.description.length >= 0)
```

Calling each methods detects following bug possibility.

- typo. - e.g. def id = get("idx") 
- field existence. (GitHub API document might be wrong.)